### ISSUE TITLE
[BUG] Datafeed links not working

### ISSUE THREAD

**Issue by susan-shu-c** [elastic/security-docs#3964]
**Description**: Summarize the problem and what needs to be omitted, edited, or replaced. 

On this page [Prebuilt jobs reference](https://www.elastic.co/guide/en/security/8.9/prebuilt-ml-jobs.html) the datafeeds seem to be formatted based on release version e.g. 8.9, so the links don't work anymore?

Link when clicked on: https://github.com/elastic/integrations/blob/8.9/packages/lmd/kibana/ml_module/lmd-ml.json#L436

A working url is this, but doesn't have the version number: https://github.com/elastic/integrations/tree/main/packages/lmd/kibana/ml_module/lmd-ml.json#L436

How can we incorporate version selection but also have a working URL?

**Doc URL link or topic name**:
<img width="1145" alt="Screenshot 2023-09-18 at 1 57 26 PM" src="https://github.com/elastic/security-docs/assets/23287722/6c917094-d9dd-49ba-9625-627a201c99c1">

**Comment by jmikell821** [comment]
Hi @terrancedejesus or @Mikaayenson - any insight to the above here?

**Comment by Mikaayenson** [comment]
> Hi @terrancedejesus or @Mikaayenson - any insight to the above here?

Good question. I dont recall us touching those files. I dont even see reference to `lmd-ml` in our code or security-docs. @ajosh0504 do you know how those links get there?

**Comment by susan-shu-c** [comment]
Those links were introduced in this `stack-docs` PR (by me): [link](https://github.com/elastic/stack-docs/pull/2496/files#diff-82af0c7ec0ef3360e221f82a47eb98997ceef9a3cfb427bc9b1d4779204160b6R433)

I see now that the `{branch}` worked during testing since it was `main` iirc but there's no `blob/[version number]` branches in `integrations` unlike the `kibana` repo

🤔 Wondering how to proceed. Maybe we can just hardcode it as `main` branch?

**Comment by Mikaayenson** [comment]
> Those links were introduced in this `stack-docs` PR (by me): [link](https://github.com/elastic/stack-docs/pull/2496/files#diff-82af0c7ec0ef3360e221f82a47eb98997ceef9a3cfb427bc9b1d4779204160b6R433)
> 
> I see now that the `{branch}` worked during testing since it was `main` iirc but there's no `blob/[version number]` branches in `integrations` unlike the `kibana` repo
> 
> 🤔 Wondering how to proceed. Maybe we can just hardcode it as `main` branch?
> 

When we do the release we commit directly to the old branches in security-docs.

**Comment by ajosh0504** [comment]
Hi! Yeah I requested @benironside to remove job and datafeed links for the Advanced Analytics jobs since they're all in a single file for any given model and we were referencing specific line numbers which could change and become hard to maintain. Instead, we link to the file containing all the configs on the docs page now. See [this](https://github.com/elastic/staging-serverless-security-docs/issues/188) PR for the changes made.

Closing this issue for now.
